scripts/gen-cli-docs.py

```diff
@@ -2,20 +2,19 @@
 Generate CLI docs.
 """
 import os
-import sys
 import argparse
-from codalab.bin import cl  # Important to put here to register the commands.
-from codalab.lib.bundle_cli import Commands
+from codalab.lib.bundle_cli import Commands, HEADING_LEVEL_1
```

Comment:
I feel like we can just use `#` here to avoid extra dependencies and make it more transparent what's going on...  If we wanted a library, it should be a separate file that both bundle CLI and this file imports, but that is probably overkill right now.

Reply:
That's true. I have removed this additional import and its related code.